src/flows/publisher_data/update_pocket_score_data_flow.py

```diff
@@ -32,53 +34,53 @@
     bq.publisher_id, bq.content_id, pcl.save_cnt, pcl.open_cnt, pcl.favorite_cnt, pcl.share_cnt 
     from `readitla_pub`.content_quality_score_beta_queue bq 
     left join `readitla_pub`.publisher_content_lifetime pcl on bq.content_id = pcl.content_id 
-          and bq.publisher_id = pcl.publisher_id limit 100
+          and bq.publisher_id = pcl.publisher_id limit 10000
 """
 
 INSERT_TO_CONTENT_V1_SQL = """
 INSERT IGNORE 
     INTO `readitla_pub`.content_quality_score (publisher_id, content_id) 
-    values ({publisher_id}, {content_id})
+    values (%s, %s)
 """
 
 INSERT_TO_CONTENT_V2_SQL = """
 INSERT IGNORE 
     INTO `readitla_pub`.content_quality_score_v2 (publisher_id, content_id) 
-    values ({publisher_id}, {content_id})
+    VALUES (%s, %s)
 """
 
 UPDATE_CONTENT_V1_SQL = """
 UPDATE `readitla_pub`.content_quality_score 
-    set save_cnt = {save_cnt}, 
-        open_uv = {open_uv}, 
-        open_dv = {open_dv}, 
-        fav_uv = {fav_uv}, 
-        fav_dv = {fav_dv}, 
-        share_uv = {share_uv}, 
-        share_dv = {share_dv}, 
-        open_score = round({open_score},4), 
-        favorite_score = round({fav_score},4), 
-        share_score = round({share_score},4), 
-        total_score = round({total_score},4) 
-        where publisher_id = {publisher_id}
-              and content_id = {content_id}
+    set save_cnt = %s, 
+        open_uv = %s, 
+        open_dv = %s, 
+        fav_uv = %s, 
+        fav_dv = %s, 
+        share_uv = %s, 
+        share_dv = %s, 
+        open_score = round(%s,4), 
+        favorite_score = round(%s,4), 
+        share_score = round(%s,4), 
+        total_score = round(%s,4) 
+        where publisher_id = %s
+              and content_id = %s
 """
 
 UPDATE_CONTENT_V2_SQL = """
 UPDATE `readitla_pub`.content_quality_score_v2 
-    set save_cnt = {save_cnt}, 
-        open_uv = {open_uv}, 
-        open_dv = {open_dv}, 
-        fav_uv = {fav_uv}, 
-        fav_dv = {fav_dv}, 
-        share_uv = {share_uv}, 
-        share_dv = {share_dv}, 
-        open_score = round({open_score},4), 
-        favorite_score = round({fav_score},4), 
-        share_score = round({share_score},4), 
-        total_score = round({total_score},4) 
-        where publisher_id = {publisher_id}
-              and content_id = {content_id}
+    set save_cnt = %s, 
+        open_uv = %s, 
+        open_dv = %s, 
+        fav_uv = %s, 
+        fav_dv = %s, 
+        share_uv = %s, 
+        share_dv = %s, 
+        open_score = round(%s,4), 
+        favorite_score = round(%s,4), 
+        share_score = round(%s,4), 
+        total_score = round(%s,4) 
+        where publisher_id = %s
+              and content_id = %s
```

Comment:
Why was this changed from named to unnamed parameters?

Reply:
The pymysql execute multiple only accepts positional arguments. :(